src/clr_bundle_add.c

```diff
@@ -45,6 +45,8 @@ static void print_help(const char *name)
 	printf("Help Options:\n");
 	printf("   -h, --help              Show help options\n");
 	printf("   -u, --url=[URL]         RFC-3986 encoded url for version string and content file downloads\n");
+	printf("   -c, --contenturl=[URL]  RFC-3986 encoded url for content file downloads\n");
+	printf("   -v, --versionurl=[URL]  RFC-3986 encoded url for version string download\n");
```

Comment:
@barbieri I'm not sure having an /etc overlay would work well for swupd... If the default config in /usr/share/defaults/swupd changed (e.g. FORMAT changed), how would updates continue to work if the /etc config is overriding the /usr config? The /etc config would not inherent the format bump, and manual intervention would be needed to ensure updates could continue past the format change.